README.md

```diff
@@ -1,2 +1,133 @@
-# notation-hashicorp-vault
-HashiCorp Vault provider for Notation CLI
+# notation-hc-vault
+HashiCorp Vault signing plugin for (Notation)[https://notaryproject.dev/]
+
+## Setup
+1. There are two binaries in the release: `key-helper` and `notation-hc-vault`.
+2. Install the `notation-hc-vault` plugin to the notation path specified by [notation plugin spec](https://github.com/notaryproject/notaryproject/blob/main/specs/plugin-extensibility.md#installation). On unix, the path is `$XDG_CONFIG_HOME/notation/plugins/hc-vault/notation-hc-vault`.
+3. `key-helper` is a standalone helper CLI used to configure Hashicorp Vault.
+
+## Install and Configure Hashicorp Vault Server
+1. [Install Hashicorp Vault](https://developer.hashicorp.com/vault/downloads)
+2. Create a vault config file under your work directory. A minimal working example is shown below, more details about config can be found [here](https://developer.hashicorp.com/vault/tutorials/operations/configure-vault#configuration-files)):
+    ```bash
+    $ cat > vault-server.hcl <<EOF
+    disable_mlock = true
+    ui            = true
+
+    listener "tcp" {
+    address     = "127.0.0.1:8200"
+    tls_disable = "true"
+    }
+
+    storage "file" {
```

Comment:
```suggestion
    storage "raft {
```

I realize the example there uses the `file` backend, but while it works for what we're doing here, it isn't an officially supported storage backend for paying customers.

Raft functions the same for this. :-)